src/frontend/Komponenter/Behandling/Vilkårpanel/Vilkårpanel.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { FC } from 'react';
+import styled from 'styled-components';
+import { Vilkårsresultat } from '../Inngangsvilkår/vilkår';
+import { Button, Heading } from '@navikt/ds-react';
+import { Collapse, Expand } from '@navikt/ds-icons';
+import { VilkårsresultatIkon } from '../../../Felles/Ikoner/VilkårsresultatIkon';
+import { BodyShortSmall } from '../../../Felles/Visningskomponenter/Tekster';
+import { ATextSubtle } from '@navikt/ds-tokens/dist/tokens';
+import {
+    EInngangsvilkår,
+    EkspandertTilstand,
+    useEkspanderbareVilkårpanelContext,
+} from '../../../App/context/EkspanderbareVilkårpanelContext';
+
+const VilkårpanelBase = styled.div`
+    margin: 2rem;
+    background-color: #f7f7f7;
+`;
+
+const VilkårpanelTittel = styled.div`
+    padding-left: 1rem;
+    display: flex;
+    justify-content: space-between;
+`;
+
+const VilkårsresultatContainer = styled.span`
+    display: flex;
+    align-items: center;
+
+    .tittel {
+        margin: 0 1rem 0 0.5rem;
+    }
+
+    .paragrafTittel {
+        color: ${ATextSubtle};
+    }
+`;
+
+interface Props {
+    tittel: string;
+    paragrafTittel?: string;
+    vilkårsresultat: Vilkårsresultat;
+    innhold: JSX.Element;
```

Comment:
Her kan man også bruke children i stedet for å sende med 1 element som et argument, som då ikke trenger å være en del av props, men kan tas med på rad 52/53

Reply:
Enig. Man må vel fortsatt definere children i props? Men fint å bruke det og heller wrappe innholdskomponentene med `Vilkårpanel`

Reply:
Sant 👍

Reply:
Her prøvde jeg å bruke `children` som prop, men slet litt med å få det til fordi `Vilkårpanel` har egne children i tillegg til de som sendes inn, og da virker det som at den slet med å skille disse. Derfor ble det lagt inn som `innhold`, i håp om at det kanskje blir tydeligere i andre komponenter at det er dette som vises i ekspandert tilstand